python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-alarm-config.json

```diff
@@ -0,0 +1,95 @@
+[
+   {
+      "EvaluationPeriods":1,
+      "ComparisonOperator":"GreaterThanThreshold",
+      "AlarmActions":[
+         "TODO: Add alarm actions! See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html"
+      ],
+      "Namespace":"CWAgent",
+      "AlarmDescription":"Memory used exceeds 90 percent for 5 minutes",
+      "Period":300,
+      "Threshold":90.0,
+      "AlarmName":"high mem_used_percent_{instance_id}",
+      "Dimensions":[
+         {
+            "Name":"InstanceId",
+            "Value":"{instance_id}"
+         }
+      ],
+      "Statistic":"Average",
+      "InsufficientDataActions":[
+
+      ],
+      "OKActions":[
+
+      ],
+      "ActionsEnabled":true,
+      "MetricName":"mem_used_percent"
+   },
+   {
+      "EvaluationPeriods":1,
+      "ComparisonOperator":"GreaterThanThreshold",
+      "AlarmActions":[
+         "TODO: Add alarm actions! See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html"
+      ],
+      "Namespace":"CWAgent",
+      "AlarmDescription":"Disk used exceeds 90 percent for five minutes",
+      "Period":300,
+      "Threshold":90.0,
+      "AlarmName":"high disk_used_percent_{instance_id}",
+      "Dimensions":[
+         {
+            "Name":"InstanceId",
+            "Value":"{instance_id}"
+         },
+         {
+            "Name":"path",
+            "Value":"/"
+         },
+         {
+            "Name":"fstype",
+            "Value":"ext4"
+         },
+         {
+            "Name":"device",
+            "Value":"nvme0n1p1"
+         }
```

Comment:
Can we get rid of these qualifiers and just add an alarm to disk usage on the root device (since not all instances are guaranteed to have an NVMe drive attached or even the same filesystem type)? If not, we may just want to remove this from our example in the interest of reducing potential maintenance overhead to keep this config working.